### DISCUSSION TITLE
What are guidelines for providing a concise descriptive alias for an AID?

### DISCUSSION THREAD

**Discussion by edeykholt** [trustoverip/kswg-keri-specification#187] [General]
UX and guidelines to suggest naming conventions for one's private alias for an AID like "Joseph as CFO for XYZ Corp". 
And/or, as publicly issued self-attestation, or as a formally structured issued credential?

**Comment by nkongsuwan** [comment]
This is not at all a trivial problem. I would like to show a good example of how NOT to name AIDs. This is my screen while I am experimenting with creating delegated multisig AIDs. The management of lots of AIDs will get confusing very quickly if their aliases are not descriptive.
![Screenshot 2567-07-03 at 21 41 25](https://github.com/trustoverip/tswg-keri-specification/assets/58167639/e07b0ebd-c7d0-461c-96c4-76307169bc1e)

**Comment by edeykholt** [comment]
There's still a conceptual issue on this, perhaps, since how a controller names their AIDs versus how they share an AID's name with others, e.g. via an OOBI connection, are quite different.  Wallets will need to work through this issue and develop best practices.
